docker/debian11/Dockerfile-jormungandr

```diff
@@ -49,6 +49,6 @@ RUN a2ensite 000-default.conf \
 
 HEALTHCHECK CMD curl -f http://localhost/v1 || exit 1
 
-EXPOSE 80
+EXPOSE 80 9091
```

Comment:
OK, so not using apache on the new health endpoint.

Reply:
I'm asking myself if you should not also specify the port `5050` which is exposed for uwsgi stats.